### ISSUE TITLE
Update gevent to the last major version

### ISSUE THREAD

**Issue by AlexBenny** [snowplow/snowplow-python-tracker#233]
Gevent dependency recently released a new major version.
we plan to update it in the next Python tracker release which is currently scheduled for Q4 2020.

**Comment by camerondavison** [comment]
I am getting
```
<frozen importlib._bootstrap>:219: RuntimeWarning: greenlet.greenlet size changed, may indicate binary incompatibility. Expected 144 from C header, got 152 from PyObject
```
which i think is caused by not being on the latest version of gevent https://github.com/python-greenlet/greenlet/issues/178 would it be worth locking down greenlet to a version <= 0.4.16 until greenlet is updated to `20.9.0` https://github.com/python-greenlet/greenlet/issues/178#issuecomment-697342964 ?

**Comment by paulboocock** [comment]
Thanks for pointing this out @camerondavison. Looks like this is in a bit of a tangle, I'll look to get this sorted out soon.

**Comment by paulboocock** [comment]
Hi @camerondavison we've just published 0.8.4 to lock greenlet to a version <= 0.4.16 until we get to bumping gevent in 0.9.0.
